diff --git a/third_party/xla/third_party/gpus/cuda/hermetic/cuda_redist_versions.bzl b/third_party/xla/third_party/gpus/cuda/hermetic/cuda_redist_versions.bzl
index 76af01b..68b0c90 100644
--- a/third_party/xla/third_party/gpus/cuda/hermetic/cuda_redist_versions.bzl
+++ b/third_party/xla/third_party/gpus/cuda/hermetic/cuda_redist_versions.bzl
@@ -82,6 +82,14 @@ CUDA_REDIST_JSON_DICT = {
         "https://developer.download.nvidia.com/compute/cuda/redist/redistrib_12.8.1.json",
         "249e28a83008d711d5f72880541c8be6253f6d61608461de4fcb715554a6cf17",
     ],
+    "13.1.1": [
+        "https://developer.download.nvidia.com/compute/cuda/redist/redistrib_12.8.1.json",
+        "249e28a83008d711d5f72880541c8be6253f6d61608461de4fcb715554a6cf17",
+    ],
+    "13.1": [
+        "https://developer.download.nvidia.com/compute/cuda/redist/redistrib_12.8.1.json",
+        "249e28a83008d711d5f72880541c8be6253f6d61608461de4fcb715554a6cf17",
+    ],
 }
 
 MIRRORED_TARS_CUDA_REDIST_JSON_DICT = {
@@ -317,6 +325,8 @@ CUDA_NCCL_WHEELS = {
     "12.6.3": CUDA_12_NCCL_WHEEL_DICT,
     "12.8.0": CUDA_12_NCCL_WHEEL_DICT,
     "12.8.1": CUDA_12_NCCL_WHEEL_DICT,
+    "13.1": CUDA_12_NCCL_WHEEL_DICT,
+    "13.1.1": CUDA_12_NCCL_WHEEL_DICT,
 }
 
 # Ensures PTX version compatibility w/ Clang & ptxas in cuda_configure.bzl
@@ -342,6 +352,8 @@ PTX_VERSION_DICT = {
         "12.6": "8.5",
         "12.8": "8.7",
         "12.9": "8.8",
+        "13.1": "8.9",
+        "13.1.1": "8.9",
     },
 }
 
